config/initializers/sentry.rb

```diff
@@ -1,10 +1,12 @@
 # frozen-string-literal: true
 
-Raven.configure do |config|
-  config.dsn = config.dsn = 'https://06e1842bac0b49fcae94e8a33ef30910@sentry.io/1859747'
-  config.current_environment = ENV['DEPLOYMENT_ENVIRONMENT'] || Rails.env
-  config.environments = %w[production test]
-  config.release = Version::VERSION
-  config.tags = { app: 'lr-dgu-ppd' }
-  config.excluded_exceptions += ['ActionController::BadRequest']
+if ENV['SENTRY_API_KEY']
```

Comment:
@ijdickinson I wonder if it wouldn't be better to check for the non-existence of the SENTRY_API_KEY and throw if it's not found? 🤔 

Otherwise LGTM... 👍

Reply:
Depends how you want to work in dev. Advantage of current approach is you can just start without have a sentry account wired up. Disadvantage is that it could lead to a production deployment where we've forgotten to set the key. Personally inclined to prioritise the former since running without sentry is not fatal.

Reply:
We definitely *don't* want Sentry logging when we're running in dev (we'd never get any peace :smile:). Tbf, there are other ways of achieving that (check the Rails.env before initialising Sentry), but istm that it's cleanest to just not init Sentry if there's no key. Once the key is sent in the deployment config, there's no reason it should change, so I don't think it's something we need routinely to test for.